activesupport/lib/active_support/testing/file_fixtures.rb

```diff
@@ -0,0 +1,35 @@
+module ActiveSupport
+  module Testing
+    # Adds simple access to sample files called file fixtures.
+    # File fixtures are normal files stored in
+    # <tt>ActiveSupport::TestCase.file_fixture_directory</tt>.
+    module FileFixtures
+      extend ActiveSupport::Concern
+
+      included do
+        class_attribute :file_fixture_directory
```

Comment:
Looking at the configuration I wonder if it'd not be confusing: `fixture_path` vs `file_fixture_directory`, as configs, considering there's also the `file_fixture_path` method. Maybe not?